src/framework/application.js

```diff
@@ -1695,6 +1695,7 @@ Object.assign(Application.prototype, {
      * @param {number|null} [settings.render.skybox] - The asset ID of the cube map texture to be used as the scene's skybox. Defaults to null.
      * @param {number} settings.render.skyboxIntensity - Multiplier for skybox intensity.
      * @param {number} settings.render.skyboxMip - The mip level of the skybox to be displayed. Only valid for prefiltered cubemap skyboxes.
+     * @param {pc.Mat3} settings.render.skyboxRotationMatrix - Rotation of skybox.
```

Comment:
Shouldn't this be:
`* @param {number[]} settings.render.skyboxRotation - Rotation of skybox.`
At least, it should be according to the example code below.

Reply:
thanks - will also have to do a conversion from array to the the object - which is actually supposed to be pc.Quat not pc.Mat3 I will fix it

Reply:
should be fixed now - thanks!